bindings-go/apis/v2/accesstypes.go

```diff
@@ -69,6 +69,46 @@ func (O *OCIRegistryAccess) SetData(bytes []byte) error {
 	return nil
 }
 
+// RelativeOciReferenceType is the access type of a relative oci reference.
+const RelativeOciReferenceType = "relativeOciReference"
+
+// RelativeOciAccess describes the access for a relative oci reference.
+type RelativeOciAccess struct {
+	ObjectType `json:",inline"`
+
+	// Reference is the relative reference to the oci image repository and tag.
+	// The format is expected to be "repository:tag".
```

Comment:
`repository@digest` should also be supported?

Reply:
afaik the tooling currently has no support for the repository@digest format, same goes for the regular ociRegistry access type